packages/messaging/ios/RNFBMessaging/RNFBMessagingModule.h

```diff
@@ -20,5 +20,5 @@
 
 
 @interface RNFBMessagingModule : NSObject <RCTBridgeModule>
-
++ (NSMutableDictionary *) initializeAppProps: (NSDictionary *)existingProps launchOptions: (NSDictionary *) launchOptions;
```

Comment:
```suggestion
+ (NSDictionary *_Nonnull) addCustomPropsToUserProps:(NSDictionary *_Nullable)userProps withLaunchOptions:(NSDictionary *_Nonnull)launchOptions;
```

Can we change it to something like this?